esmvalcore/cmor/_fixes/cesm/cesm2.py

```diff
@@ -49,3 +49,27 @@ def fix_metadata(self, cubes):
         self.fix_var_metadata(cube)
 
         return CubeList([cube])
+
+    def _fix_time(self, cube):
+        """Fix time coordinate of cube.
+
+        For monthly data that does not correspond to point measurements, move
+        time points to center of time period given by time bounds (currently
+        the points are located at the end of the interval).
```

Comment:
please provide this in the docstring: an example of old data, and example of new data as per desired behaviour

Reply:
Done :+1: